config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.1/deployment.yaml

```diff
@@ -109,3 +109,14 @@ items:
           repository: k8s.gcr.io/sig-storage/livenessprobe
           tag: "v2.6.0"
           imagePullPolicy: IfNotPresent
+  - apiVersion: v1
+    kind: secret
+    metadata:
+      name: "{{secret-name}}"
```

Comment:
If this name is provided by user, how template resources/operator would know which secret need to look into for credentials?
Shouldn't be secret name specified as per your plugin?

Reply:
the user creates the [storageclass](https://github.com/ArbelNathan/ibm-satellite-storage/blob/424cc3a684067151990b6e62beb357259706bbde/config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.1/storage-class-template.yaml#L18) that points to this secret. we don't need it anywhere else.